sdk/nodejs/cmd/run/run.ts

```diff
@@ -117,7 +117,7 @@ function reportModuleLoadFailure(program: string, error: Error): never {
     return process.exit(1);
 }
 
-export function run(argv: minimist.ParsedArgs): void {
+export function run(argv: minimist.ParsedArgs, programStarted: () => void): void {
```

Comment:
Maybe I'm being dense but I'm not seeing any corresponding change to callers of `run` to pass a function - what's the semantics here?